config/pipelines/high_throughput_scrna_core_cdna_prep.wip.yml

```diff
@@ -17,10 +17,10 @@ scRNA Core cDNA Prep:
     LRC GEM-X 5p Chip: LRC GEM-X 5p GEMs
     LRC GEM-X 5p GEMs: LRC GEM-X 5p cDNA PCR
     LRC GEM-X 5p cDNA PCR: LRC GEM-X 5p cDNA PCR XP
-scRNA Core cDNA Prep Input:
+scRNA Core cDNA Prep GEM-X 5p PBMC Pools Input:
   pipeline_group: scRNA Core cDNA Prep
   filters:
-    request_type_key: limber_scrna_core_cdna_prep_input
+    request_type_key: limber_scrna_core_cdna_prep_gem_x_5p_pbmc_pools_input
```

Comment:
Same comment here as in the Sequencescape PR - there's a bit of inconsistency as to whether 'pbmc_pools' is included or not.